lib/screens/NavigationWrapper.dart

```diff
@@ -33,12 +34,13 @@ class _NavState extends State<Nav> {
       ),
     ),
     ReliefHomeScreen(),
-    Center(
-      child: Text(
-        'Me',
-        style: TextStyle(fontSize: 60),
-      ),
-    )
+    ProfileScreen(),
+    // Center(
```

Comment:
Would you mind removing this commented code? Unless you have a good reason for keeping it here :)